Add warning about construction work in docs (#1020)
zkSNACKs/WasabiDoc: README.md

<p align="center">
  <a href="https://docs.wasabiwallet.io">

>     <img src="docs/.vuepress/public/WasabiWalletDocs.png" alt="Wasabi Wallet Documentation" title="Wasabi Wallet Documentation" class="Image" />

Reviewer: When replacing a file or a picture, make sure it gets deleted if it is not used elsewhere.

>     <img src="docs/.vuepress/public/Under_Construction.png" alt="Documentation under construction" title="Documentation under construction" class="Image" />

Reviewer: I don't think we should change the tags:

```suggestion
    <img src="docs/.vuepress/public/Under_Construction.png" alt="Wasabi Wallet Documentation" title="Wasabi Wallet Documentation" class="Image" />
```

  </a>
</p>

> ## Note that [Wasabi Wallet documentation](https://docs.wasabiwallet.io/) is under renovation, which means some content you may want to work on is about to be obsoleted and some of the information you see in this documentation might not be up to date. 

Reviewer: ```suggestion
### Note that [Wasabi Wallet documentation](https://docs.wasabiwallet.io/) is under renovation, some content is about to be obsoleted and some information might not be up to date.
```

> [Wasabi Wallet](https://wasabiwallet.io) is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop. It implements a trustless privacy technique, called [WabiSabi CoinJoin](https://eprint.iacr.org/2021/206).

Reviewer: How about this?

```suggestion
[Wasabi Wallet](https://wasabiwallet.io) is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop. It implements a trustless privacy technique, called [CoinJoin](https://en.bitcoin.it/Privacy#CoinJoin).
```
